packages/web3/integrationTests/SendTransaction.Itest.js

```diff
@@ -0,0 +1,15 @@
+import Web3 from 'web3';
+
+describe('Integration test for sending ether', () => {
+    it('Should successfully resolve on sending ether', async () => {
+        const web3 = new Web3('http://ganache:8545', undefined, {transactionConfirmationBlocks: 1});
+        const [sender, recipient] = await web3.eth.getAccounts();
+        await expect(
+            web3.eth.sendTransaction({
+                value: '1',
+                from: sender,
+                to: recipient
+            })
+        ).resolves;
```

Comment:
I am not sure that `await expect(myPromise).resolve` does exactly what I intend to do: Wait for the promise to resolve and ignore the returned value. Does anyone know more?

Reply:
``` javascript 
await expect(
            web3.eth.sendTransaction({
                value: '1',
                from: sender,
                to: recipient
            })
        ).resolves.toEqual(...);
``` 

Would be the correct way to test the Promise case and the EventEmitter case has to get added to. 

``` javascript 
it('Should successfully resolve on sending ether', (done) => {
        const web3 = new Web3('http://ganache:8545', undefined, {transactionConfirmationBlocks: 1});
        const [sender, recipient] = await web3.eth.getAccounts();
        web3.eth.sendTransaction({
            value: '1',
            from: sender,
            to: recipient
        })
        .on('confirmation', (confirmation) => { .... });
        .on('receipt', (receipt) => { expect(receipt).toEqual(...); done();  });
});
```

Reply:
https://jestjs.io/docs/en/expect

Reply:
I wanted to add one simple test. You can later add more tests if you want.

Also I do not want to verify the value returned by the promise. I only want to make sure the promise resolves. Right now I include just a very very simple test to show how it works. Please do not make the PR more complex by asking for more tests. They can be added easily later.

Reply:
I think it works as is btw because I tested with a non-working url for the ganache server and the test failed.

Reply:
Maybe there is something like
```js
await expect(
            web3.eth.sendTransaction({
                value: '1',
                from: sender,
                to: recipient
            })
        ).resolves.toEqual(iDontGiveAF***);
```

Reply:
``` javascript
 await web3.eth.sendTransaction({
                value: '1',
                from: sender,
                to: recipient
            })
```

Is enough because the test will fail if the Promise takes to long or if it rejects with an error.

Reply:
That is not a test. If this fails you will never get proper information on exactly which test failed.

Reply:
It will show you the description string and the failed state. We could also just expect the properties from the receipt which aren't generic. 

``` javascript 
await expect(web3.eth.sendTransaction({
  value: '1',
  from: sender,
  to: recipient
})).resolves.toHaveProperty('status', true);
```

But I think it would be better if we could expect the complete receipt object with ``toEqual()``.

Reply:
Please make this a suggestion next time like this:
```suggestion
        ).resolves.toHaveProperty('status', true);;
```